### PR TITLE
update diamond package

### PR DESCRIPTION
**Pull request by bgruening**
Diamond 0.7.9 was not uploaded to the MTS so I replaced this version with a new one.
Ping @nekrut

### CODE REVIEW

File: packages/package_diamond_0_7_10/tool_dependencies.xml [hunk 1]
```diff
@@ -1,11 +1,11 @@
 <?xml version="1.0"?>
 <tool_dependency>
-    <package name="diamond" version="0.7.9">
+    <package name="diamond" version="0.7.10">
         <install version="1.0">
             <actions_group>
                 <!-- Download the binaries for BLAST+ compatible with 64-bit Linux. -->
                 <actions os="linux" architecture="x86_64">
-                    <action md5sum="d3f23e358891f7d08b1fcd558eb294ee" type="download_by_url">https://github.com/bbuchfink/diamond/releases/download/v0.7.9/diamond-linux64.tar.gz</action>
+                    <action sha256sum="db6cf64839b989306cfba752c05c192acc1bce133e0821a63e70a882d22b96aa" type="download_by_url">https://github.com/bbuchfink/diamond/releases/download/v0.7.10/diamond-linux64.tar.gz</action>
```
Comment (erasche) [inline packages/package_diamond_0_7_10/tool_dependencies.xml]: thanks!

**Comment by rec3141** [comment]
existing diamond doesn't work because the syntax changed in version 0.7.0

this may require a new tool for `diamond view`

> Important notice for update 0.7.0:
> This update changes the syntax for calling the program and introduces a new file format (DAA) for storing alignments. Other formats are generated using the view command on a DAA file. This is to reduce file sizes and to allow generating several different output formats from the same alignments.
> Example invocation for aligning a file reads.fna, saving the results in matches.daa and then generating output in compressed SAM format:
> $ diamond blastx -d nr -q reads.fna -a matches -t~
> $ diamond view -a matches.daa -o matches.sam -f sam --compress 1`

**Comment by bgruening** [comment]
@rec3141  This is strange the tool should still depend on 0.6.13. I never got around to update the wrapper with the new output format :(
Nevertheless, work started here: https://github.com/bgruening/galaxytools/pull/281

Do you have a concrete error message? Which argument was not found?
Thanks!

**Comment by rec3141** [comment]
I don't quite understand the galaxy toolshed, but it looks like the package that installs diamond downloads the most recent version available, rather than a specific version e.g. '0.6.13'

https://toolshed.g2.bx.psu.edu/repository?repository_id=02085af63a6e1cc8

If this package has been run in the past year or so, then it would have overwritten v0.6.13

**Comment by nsoranzo** [comment]
@rec3141 is right, https://toolshed.g2.bx.psu.edu/view/iuc/package_diamond_0_6_13/ is downloading from http://www-ab.informatik.uni-tuebingen.de/data/software/diamond/download/public/diamond-linux64.tar.gz which when run says: "diamond v0.7.2.51" . Newer versions (from 0.7.3) are released on GitHub https://github.com/bbuchfink/diamond/

**Comment by martenson** [comment]
This means some people have old wrapper and new binary that are incompatible. What is a good way to remedy this?  Update both? 

We probably have no good means to update past version of wrapper.

**Comment by gregvonkuster** [comment]
I think the only fix is to update both the wrapper and the dependency install recipe to require a static version.

**Comment by nsoranzo** [comment]
And then deprecate `package_diamond_0_6_13` on the Tool Shed.

**Comment by bgruening** [comment]
@rec3141 can you try to update your package_diamond and see if this fixes your problem?

**Comment by rec3141** [comment]
Thank you. I'm using the main Galaxy instance, not my own installation.

**Comment by SilviaRaineri** [comment]
Hello! Me and others are experiencing the same problem (diamond-makedb is working, but the Diamond tool itself isn't - at least, not in the main Galaxy instance). Whenever I try to run a job, the output says **insufficient arguments, use diamond  -h for help** (see attachment). Here is what I found regarding the version of the tool:
Galaxy Tool ID: toolshed.g2.bx.psu.edu/repos/bgruening/diamond/bg_diamond/0.1.6.0 Galaxy Tool (but when you try to reach this page, it gives an error)
Version: 0.1.6.0 This tool requires diamond (Version 0.6.13)
Job dependencies diamond tool_shed_package 0.6.13

I know this problem has been discussed before, but as I am using the main Galaxy site I don't know how to update this tool. Can anyone help?

screenshot from 2017-05-16 11-20-34
Best, 
Silvia

**Pull request merged**